Src/Base/AMReX_TimeIntegrator.H

```diff
@@ -83,6 +115,9 @@ public:
             case IntegratorTypes::ExplicitRungeKutta:
                 integrator_ptr = std::make_unique<RKIntegrator<T> >(S_data);
                 break;
+            case IntegratorTypes::Sundials:
+                integrator_ptr = std::make_unique<SundialsIntegrator<T> >(S_data);
+                break;
```

Comment:
Does this case need `#ifdef AMREX_USE_SUNDIALS`?

Reply:
Good catch, done!